front-end/src/App.js

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React from "react";
+import Profile from "./Components/Profile.js";
+import Home from "./Components/Home.js";
+import Navbar from "./Components/Navbar.js";
+import Sidebar from "./Components/Sidebar.js";
```

Comment:
lets seperate the idea of components and pages

Navbar and Sidebar are components
Profile and Home are pages

so you should probably make a pages directory and a components directory.

Reply:
Okay, I just made these changes.